### ISSUE TITLE
“Close” graphical buttons with insufficient accessible names [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#240]
The close ("X") button that closes the "Choose your county" modal in the "health About" screen is announced by TalkBack as 'close choose, close choose, button". The information is repeated (redundant) and does not clearly indicate which modal it closes (e.g. 'close, choose your county modal, button").

![health About - Close x Choose](https://user-images.githubusercontent.com/7034772/92030583-3ff08b80-ed35-11ea-9bff-499632cd3da3.png)

**Remediation Guidance:**
Remove the redundant "close choose" text and include "close, choose your county modal".

Severity: Medium

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed

**Comment by jonnyjames** [comment]
This issue is now fixed. I am closing this issue.
